src/write.rs

```diff
@@ -515,7 +512,8 @@ impl ZipWriterStats {
 impl<A: Read + Write + Seek> ZipWriter<A> {
     /// Initializes the archive from an existing ZIP archive, making it ready for append.
     pub fn new_append(mut readwriter: A) -> ZipResult<ZipWriter<A>> {
-        let (footer, cde_start_pos) = spec::CentralDirectoryEnd::find_and_parse(&mut readwriter)?;
+        let (footer, cde_start_pos) =
+            spec::Zip32CentralDirectoryEnd::find_and_parse(&mut readwriter)?;
```

Comment:
No: `CentralDirectoryEnd` to `Zip32CentralDirectoryEnd` was a mechanical change I performed via regex because it felt unclear to me which operations were searching for zip32 central directory headers and which for zip64 data. If we'd prefer to avoid this rename, I can absolutely revert it.